Myslot.lua

```diff
@@ -752,6 +752,14 @@ function MySlot:RecoverData(msg, opt)
                             end
                         end
 
+                        -- another fallback option - try to get base spell
+                        if not GetCursorInfo() then
```

Comment:
```suggestion
                        if not GetCursorInfo() and FindBaseSpellByID then
```

work with client prior to 10.0


thanks, actually, i do not no this api, what i did, as you can see, is to create a `CreateSpellOverrideMap`

Reply:
did you mean to suggest the change above?

Reply:
yup, let me know if it looks good to you

Reply:
ahh i see what you mean, lgtm ill give it a quick test today